Source/CarthageKit/Project.swift

```diff
@@ -1064,6 +997,21 @@ public final class Project { // swiftlint:disable:this type_body_length
 					return dependenciesIncludingNext
 				}
 			}
+			.flatMap(.concat) { (dependencies: [(Dependency, PinnedVersion)]) -> SignalProducer<[(Dependency, PinnedVersion)], CarthageError> in
+				return self.copyBuiltFrameworksFromSharedCacheIfVersionMatches(for: dependencies, options: options)
```

Comment:
I think copying to/from the shared cached should be hidden behind another flag. We already have the in-repo cache behind a flag and this is inherently riskier.

Reply:
added a `if options.cacheBuilds {` block to prevent copy from shared cache without `--cache-builds` being set